### PR TITLE
improve contributing section

### PR DESCRIPTION
**Pull request by Tlazypanda**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/contributing/ways_to_contribute.rst [hunk 1]
```diff
@@ -1,63 +1,90 @@
...
+* Join moja global by emailing on info@moja.global. You will receive a slack workspace invite where you may join us for further discussions on the project of your interest.
+* You can review and answer contributor questions on open issues in your repository/team repositories on our ` organization GitHub <>`_.
+* You may also review discussions and answer new users queries on moja global's Slack workspace
+* If the reply could be useful for others, please add it as an entry under the FAQ section on the README of your repo or in this documentation repo FAQ section for general moja global questions.
+* Don't forget to `Claim credit <index.html#get-credit-for-your-contribution>`_ for your contribution
+
+Organize moja global events/meetups
+===================================
+
+moja global believes events are important to reach out to new users or improve collaboration. If you are planning to organize events to spread the message of moja global and invites contributors, here are the steps to follow:
```
Comment (priyanka-choubey) [inline docs/contributing/ways_to_contribute.rst:77]: please change "invites" to "invite"

File: docs/contributing/ways_to_contribute.rst [hunk 2]
```diff
@@ -1,63 +1,90 @@
...
+* Add your name to Coaches/Ambassadors in the README document by copying your avatar and name from the ``all-contributors`` section and pasting it under the **Coaches**/**Ambassadors** header.
+* Submit a Pull-Request with your proposed changes
+
+Answer user Questions
+=====================
+
+Please help moja global be an active and responsive open source organization! Here is how you may answer user queries.
+
+* Join moja global by emailing on info@moja.global. You will receive a slack workspace invite where you may join us for further discussions on the project of your interest.
+* You can review and answer contributor questions on open issues in your repository/team repositories on our ` organization GitHub <>`_.
```
Comment (priyanka-choubey) [inline docs/contributing/ways_to_contribute.rst:69]: redirection url for "organization github" not added here

File: docs/contributing/index.rst [hunk 3]
```diff
@@ -8,33 +8,61 @@ We're so glad that you're thinking about contributing to moja global. We welcome
...
+  code_contribution_best_practices
+  coc
+  ways_to_contribute
 
-There are plenty of ways you can contribute to moja global repositories. To know more about how to contribute please click `here <ways_to_contribute>`_.
+Before making a contribution
+----------------------------
 
-**After making your first contribution**
+* Please checkout our `Git and Github guide <../DevelopmentSetup/git_and_github_guide.html>`_ on detailed instructions on how to setup this project and make a pull request.
```
Comment (priyanka-choubey) [inline docs/contributing/index.rst:23]: suggestion to change "on detailed instructions" to "for detailed instructions"

File: docs/contributing/index.rst [hunk 4]
```diff
@@ -8,33 +8,61 @@ We're so glad that you're thinking about contributing to moja global. We welcome
...
+Review other contributions
+==========================
+
+moja global welcomes all contributors to review each others pull requests and suggest changes. You may also choose to review other contributions by following this guide `here <../DeveloperWorkflow/reviewing.html>`_.
+
+Non-code contributions to moja global
+-------------------------------------
+
+There are plenty of ways you can contribute to moja global repositories.
+Apart from contributing code, moja global would love to any kind of help for non-code related issues as well.
```
Comment (priyanka-choubey) [inline docs/contributing/index.rst:66]: Change this line to "moja global would love any kind of help with non-code related issues as well"

File: docs/DevelopmentSetup/git_and_github_guide.rst [hunk 5]
```diff
@@ -36,18 +36,18 @@ Fork and Clone this project
...
+To work on an issue, claim it by adding a comment with ``@botmojaglobal claim`` to the issue thread. botmojaglobal is a GitHub workflow bot forked from the `zulipbot <https://github.com/zulip/zulipbot/>`_; it will assign you to the issue and label the issue as ``in progress``. Some additional notes:
 
-* You can only claim issues with the `Good for newcomers` or `Help Wanted` labels. botmojaglobal will give you an error if you try to claim an issue without one of those labels.
+* You can only claim issues with the ``Good for newcomers`` or ``Help Wanted`` labels. botmojaglobal will give you an error if you try to claim an issue without one of those labels.
 
-* Please feel free to ask questions on how to approach the issue or if the tests are failing. The maintainers/reviewers will try to get back to you as soon as possible. You can reach us on the moja-global `slack <mojaglobal.slack.com>`_`, or through Github.
+* Please feel free to ask questions on how to approach the issue or if the tests are failing. The maintainers/reviewers will try to get back to you as soon as possible. You can reach us on the moja-global `slack <mojaglobal.slack.com>`_, or through Github.
```
Comment (priyanka-choubey) [inline docs/DevelopmentSetup/git_and_github_guide.rst:50]: The slack link seems to be inactive.

**Review by priyanka-choubey — COMMENTED** [review]
I have commented a few suggestive changes and at a couple of faulty redirection URLs. Other than this, the documentation seems good to me.

**Comment by Tlazypanda** [comment]
Hey @priyanka-choubey Thanks for the review have made the changes :v: can you review again?

**Review by priyanka-choubey — APPROVED** [review]

**Comment by aornugent** [comment]
This looks great, thanks! Closes #19. Well done!

**Comment by Tlazypanda** [comment]
Thanks for the review @aornugent :tada:

**Pull request merged**